d2l-table-style.html

```diff
@@ -62,24 +62,17 @@
 				@apply(--d2l-table-body);
 			}
 
-			:host > ::content > * > tr,
-			:host > ::content > tr {
+			:host > ::content > * > tr {
 				@apply(--d2l-table-row);
 			}
 
-			:host > ::content > thead > tr > th,
 			:host > ::content > * > tr > td,
-			:host > ::content > tr > td,
-			:host > ::content > tbody > tr > th,
-			:host > ::content > tfoot > tr > th {
+			:host > ::content > * > tr > th {
 				@apply(--d2l-table-cell);
 			}
 
-			:host-context([dir="rtl"]) > ::content > thead > tr > th,
 			:host-context([dir="rtl"]) > ::content > * > tr > td,
-			:host-context([dir="rtl"]) > ::content > tr > td,
-			:host-context([dir="rtl"]) > ::content > tbody > tr > th,
-			:host-context([dir="rtl"]) > ::content > tfoot > tr > th {
+			:host-context([dir="rtl"]) > ::content > * > tr > th {
```

Comment:
(for this case where the structure is pretty understood, and we're not accidentally including 3000 more elements in the search)